.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: rust
 sudo: false
-rust:
-  - nightly
-env:
-  - FEATURES=""
-  - FEATURES="unstable"
+matrix:
+  include:
+    - rust: nightly
+    - rust: nightly
+      env:
+      - FEATURES="unstable"
 script: |
-  cargo build --verbose --features "${FEATURES}"
+  cargo build --verbose --features "${FEATURES}" &&
   cargo test --verbose --features "${FEATURES}"
 after_success: "./deploy_gh_pages.sh"
```

Comment:
And yeah, this could actually cause multiple deploys, but it hasn't been a problem, yet (fingers crossed!).